### PR TITLE
Add way for users to be able to provide feeback

### PR DESCRIPTION
**Pull request by Rytiggy**

### CODE REVIEW

**Comment by Rytiggy** [comment]
screen shot 2017-09-21 at 4 07 00 pm
screen shot 2017-09-21 at 4 06 50 pm

File: app/components/user-feedback-form/template.hbs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{{yield}}
+<buttin class='btn btn-default user-feedback' {{action 'toggleForm'}} ><div>Provide feedback</div></buttin>
```
Comment (birdbrained) [inline app/components/user-feedback-form/template.hbs]: button?

Comment (Rytiggy) [reply app/components/user-feedback-form/template.hbs]: good catch

File: app/styles/app.css [hunk 2]
```diff
@@ -318,4 +318,80 @@ pre {
...
 .popover-button i{
   color:#666;
-}
\ No newline at end of file
+}
+
+
+.user-feedback{
+  position:fixed;
+  bottom:100px;
```
Comment (birdbrained) [inline app/styles/app.css]: A thought; is this ok as a fixed distance or is there value in using a percentage here?

Comment (Rytiggy) [reply app/styles/app.css]: I don't think that having a percentage would change much its just the button sitting on the side of the screen

Comment (birdbrained) [reply app/styles/app.css]: Also it's pretty far down the page, what do we think users would do if it were at 50% or further up? @cameronblandford thoughts?

Comment (cameronblandford) [reply app/styles/app.css]: I think having it most of the way down the page keeps it from being too distracting, and as soon as the user scrolls even a tiny bit, they'll absolutely notice it. I think it's obvious but not detracting, where it is. I wouldn't move it

Comment (birdbrained) [reply app/styles/app.css]: Ok cool.

**Comment by NyanHelsing** [comment]
Looks good! merging in

**Pull request merged**